### PR TITLE
Create weather

### PR DESCRIPTION
**Pull request by AnhMai-bit**

### CODE REVIEW

File: weather
Comment (ufshk) [inline weather:1]: ❗ 
Shouldn't be committing the API key directly. Have a separate config file that you import from. The config file should not be committed either since it would contain the key. 

See the routemodel/data_retrieval folder in the repository for an example of how we keep the API key secure.

**Review by ufshk — CHANGES_REQUESTED** [review]
Resolve the comment and regenerate the API key (without committing it again) and you should be good! 

I'll let Emma and Shio approve this since they have the requirements for the project, dismiss my review when you've done the above.

File: weather
Comment (ufshk) [inline weather:4]: We should take care to make our code a bit more portable. Try to use a relative path, or receive the inputs from the user in another way.

File: weather
Comment (ufshk) [inline weather:3]: Are these headers used? It seems to be empty for the entire file, only the key 'Content-Type' is there?

File: weather
Comment (ufshk) [reply weather:4]: I'm also not sure if this is referring to a file or not. Doublecheck to see if this works the way you intend it to!

**Comment by e-wai** [comment]
Using https://github.com/uw-midsun/strategy/pull/110 instead

**Pull request closed without merging**
